tests/platforms/android/apk/test_verify_tools.py

```diff
@@ -0,0 +1,183 @@
+import subprocess
+from io import BytesIO
+from stat import S_IMODE
+from unittest import mock
+from zipfile import ZipFile
+
+import pytest
+from requests import exceptions as requests_exceptions
+
+from briefcase.exceptions import BriefcaseCommandError, NetworkFailure
+from briefcase.platforms.android.apk import ApkBuildCommand
+
+
+def create_sentinel_zipfile():
+    out = BytesIO()
+    with ZipFile(out, "w") as zipfile:
+        zipfile.writestr("tools/bin/sdkmanager", "")
+    return out.getvalue()
+
+
+@pytest.fixture
+def build_command(tmp_path, first_app_config):
+    class TestableApkBuildCommand(ApkBuildCommand):
+        dot_briefcase = tmp_path / ".briefcase" / "tools"
+
+    command = TestableApkBuildCommand(
+        base_path=tmp_path / "base_path", apps={"first": first_app_config},
+    )
+    command.tmp_path = tmp_path
+    command.host_os = "Linux"
```

Comment:
Is this necessary? If it is, general preference would be to set it to a value that *can't* exist, so it catches any case where we're implicitly assuming the OS.

Reply:
Fair enough!

Reply:
+1, I'll use a sentinel value.

Reply:
Fixed in latest push